### DISCUSSION TITLE
Why is .>>. giving an F# error

### DISCUSSION THREAD

**Discussion by jcoder58** [teo-tsirpanis/Farkle#130] [Q&A]
First time Farkle (6.5.1) user.  Why am I getting a compile error in the following Farkle grammar?

    open Farkle.Builder
    module rec Grammar =
    let hex_quad = "hex_quad" ||= [
        !@ hexadecimal_digit .>>. hexadecimal_digit .>>. hexadecimal_digit .>>. hexadecimal_digit
    ]
    let universal_character_name = "universal_character_name" ||= [
        !& @"\u" .>>. hex_quad
       // Error: None of the types 'ProductionBuilder, DesigntimeFarkle<'a>' support the operator '.>>.'	
        !& "\U" .>>. hex_quad .>>. hex_quad
    ]

**Comment by teo-tsirpanis** [comment]
Hi @jcoder58 and thanks for using Farkle!

Can you provide a more complete example? I cannot build the attached snippet. Here's some feedback from what I can understand:

* Are you using `universal_character_name` as part of a string? The `Terminals.string` and `stringEx` function might be of interest to you and it is much more performant.
* At the end of building a production, you have to finish it with the `=>` operator.

Let me know if you have more questions.

**Reply by teo-tsirpanis** [reply]
@jcoder58 I got an email that you sent a reply which looks like you later deleted. Does that mean that your problem is solved?
